src/message.cpp

```diff
@@ -93,6 +93,19 @@ struct KeywordMap SimpleKeywords[] = {
 
 #define KEYWORD_SIZE 256
 
+static char* quoted_strchr(char* s, int c)
+{
+    char* p;
```

Comment:
Can't, since the function consuming is expects `char*`. And that can't be fixed since it modified the string (adds `\0`s).